docs/commands/changeset-derive.asciidoc

```diff
@@ -7,14 +7,12 @@ OSM datasets.  Alternatively, when only one dataset is input it generates a chan
 the data in the input dataset. The output changeset file can later be applied to an OSM API database with 
```

Comment:
I removed support for multiple outputs during a single changeset derivation (e.g. a .osc and and an .osc.sql).  This feature wasn't actually being used, and made the changeset-derive code a lot nastier if we wanted to avoid multiple passes over the data during output.